src/dmd/mtype.d

```diff
@@ -6655,16 +6655,18 @@ extern (C++) final class TypeTag : Type
     Type resolved;          /// type after semantic() in case there are more others
                             /// pointing to this instance, which can happen with
                             ///   struct S { int a; } s1, *s2;
+    MOD mod;                /// modifiers to apply after type is resolved (only MODFlags.const_ at the moment)
```

Comment:
Add this to mtype.h as well.

Reply:
TypeTag is only a stub in mtype.h.

Reply:
Is it not integrated into the visitor class?